### PR TITLE
Fix #2 Default upload file size limit

### LINKED ISSUE #2: [BUG] Files more than 100mb not uploaded
**Describe the bug**
Files more than 100mb not uploaded with an error:
```
panic: put command failed on CloseAndRecv: rpc error: code = Unknown desc = objectService.Put failed: incoming object has not passed filters: filter OBJECT_SERVICE_PUT_CHECKPOINT finished with result FAILED [] {subfilter 1: filter PLUGINS_CHECKPOINT_FILTER finished with result IGNORED [no sub result => auto ignore] {subfilter 1: filter NEOFS_PLUGIN_FOR_OBJECT_SERVICE finished with result IGNORED [object is not a NeoFS plugin] {no sub filters}}, subfilter 2: filter OBJECTS_OVERALL finished with result FAILED [] {subfilter 1: filter INTEGRITY finished with result PASSED [object has valid user token] {no sub filters}, subfilter 2: filter OBJECT_SIZE finished with result FAILED [object size 134270164 overflows max value 104857600] {no sub filters}}}

goroutine 1 [running]:
main.main()
        /src/cmd/cli/main.go:267 +0x24c

```
But there's no comment in documentation about any limitations.

**To Reproduce**
1. `make local_up`
2. `make local_cli`
3. `neofs-cli container put --rule 'SELECT 3 Node'`
4. `object put --cid 3QEhR3QMDcRvRCBdA4DLfB3Se2hyJUVGvLKSukH4CTxc --file /data/filename.mp4` (filename.mp4 size is over 100mb)

**Expected behavior**
Comment in README about max file size or having no limitation.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 10.14.6 (18G95)

### PR DESCRIPTION
**Pull request by im-kulikov**
- add env variable
- add Readme description

### CODE REVIEW

File: neofs-local/README.md [hunk 1]
```diff
@@ -5,22 +5,39 @@
...
 *Contains pregenerated private keys for IR nodes*
 
-## Run environment
+## FAQ:
+
+### Configuration
+
+**Increase default upload file size limit:**
+
+By default, in .env you can find `NEOFS_OBJECT_MAX_PROCESSING_SIZE=300` (**300MB**),
```
Comment (fyrchik) [inline neofs-local/README.md]: It is set in `.env` by `NEOFS_OBJECT_MAX_PROCESSING_SIZE` (**300MB** by default). You can override this value.

**Review by fyrchik — APPROVED** [review]

**Review by anatoly-bogatyrev — APPROVED** [review]

File: .env [hunk 2]
```diff
@@ -15,6 +15,7 @@ PROMETHEUS_VERSION=2.12.0
 
 # NeoFS images versions
 NEOFS_IMAGES_VERSION=0.0.5
+NEOFS_DROPUI_VERSION=0.0.2-pre
```
Comment (volekerb) [inline .env:18]: I would prefer to have this change in separate commit since it's not related to #2

File: neofs-local/docker-compose.drop.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ services:
       - 7001:80
 
   drop-ui:
-    image: nspccdev/neofs-drop-ui:0.0.2-pre
+    image: nspccdev/neofs-drop-ui:${NEOFS_DROPUI_VERSION}
```
Comment (volekerb) [inline neofs-local/docker-compose.drop.yml:16]: this once as well in separate commit

File: neofs-local/README.md [hunk 4]
```diff
@@ -5,22 +5,39 @@
...
 
 ```
 $ make local_cli
 ```
 
-*You can use `holder01:8080` as host address for connection*
\ No newline at end of file
+*You can use `holder01:8080` as host address for connection*
+
+## Run NeoFS DropIn service
```
Comment (volekerb) [inline neofs-local/README.md:37]: And this part as well

File: .env (hunk 2 above)
Comment (im-kulikov) [reply .env:18]: done

File: neofs-local/README.md (hunk 4 above)
Comment (im-kulikov) [reply neofs-local/README.md:37]: done

**Pull request merged**